src/cljs/rems/application.cljs

```diff
@@ -509,20 +506,26 @@
 (defn- save-button []
   [atoms/rate-limited-button {:id "save"
                               :text (text :t.form/save)
-                              :disabled @(rf/subscribe [:rems.spa/pending-request ::save-draft])
+                              :disabled @(rf/subscribe [:rems.spa/any-pending-request #{"/api/applications/save-draft"
+                                                                                        "/api/applications/submit"
+                                                                                        "/api/applications/copy-as-new"}])
                               :on-click #(rf/dispatch [::save-application [text :t.form/save]])}])
 
 (defn- submit-button []
   [atoms/rate-limited-button {:id "submit"
                               :text (text :t.form/submit)
                               :class :btn-primary
-                              :disabled @(rf/subscribe [:rems.spa/pending-request ::submit])
+                              :disabled @(rf/subscribe [:rems.spa/any-pending-request #{"/api/applications/save-draft"
+                                                                                        "/api/applications/submit"
+                                                                                        "/api/applications/copy-as-new"}])
```

Comment:
This is OK but perhaps a bit fragile when we add new commands. Would it actually be simpler if we would always just wait for one thing? So that every action blocks the others? Maybe for application? 🤔